prover/lang/kore-lang.md

```diff
@@ -778,6 +797,13 @@ Simplifications
   rule hasImplicationContext(\typeof(P, _))
     => hasImplicationContext(P)
 
+  syntax Pattern ::= "#\\exists" "{" Patterns "}" Pattern [function]
+  rule #\exists{.Patterns} P => P
+  rule #\exists{V, Vs} P => \exists{V, Vs} P
+
+  syntax Pattern ::= "#\\and" "(" Patterns ")" [function]
```

Comment:
Not sure about hyperlogics, but for SL, we've found it easier to standardize on a normal form where `\exists`, `\and` etc are always present in a formula irrespective of whether it is needed. The normal form we use is `\implies(\and(...), \exists {... } \and(...))`